README.md

```diff
@@ -134,7 +134,8 @@ For a good example, see https://riot.im/develop/config.json
 1. `integrations_rest_url`: URL to the REST interface for the integrations server.
 1. `integrations_widgets_urls`: list of URLs to the REST interface for the widget integrations server.
 1. `bug_report_endpoint_url`: endpoint to send bug reports to (must be running a
-   https://github.com/matrix-org/rageshake server)
+   https://github.com/matrix-org/rageshake server). Bug reports can be disabled by leaving the 
+   `bug_report_endpoint_url` out of your config file.
```

Comment:
I wonder if we should add some sort of text here to say that bug reports aren't automatic.

Reply:
Something like "Bug reports are sent when user click "Send Logs" within the application" is probably okay.